README.md

```diff
@@ -193,124 +200,167 @@ Other acceptable location examples:
 - as396982, as16509
 - comcast, google, ovh, orange, t-mobile
 
-### Stack filters
+##### Flags
+All network test commands share some flags but also have unique ones. To learn more, run `--help` with the respective test type. For example, to find out more about `ping`, run:
 
-You can also combine multiple locations and use them as filters to increase the accuracy of your tests. For this purpose we use the plus symbol +.
+`globalping ping --help`
 
-- `from comcast+california` will return a probe that is both hosted with Comcast and located in CA.
-- `from google+europe` will return a random probe hosted with Google Cloud in Europe, or if you want to be more specific but dont remember the cloud region name:
-- `from google+germany` will return a probe with Google Cloud Germany
+###### The limit flag
+The `--limit` flag is probably one of the most useful flags when starting with Globalping. It determines the number of tests to run, which is set to one (1) by default. 
 
-You can combine as many parameters as you want too
+For example:
+- `from north america` selects one probe in either USA or Canada.
+- `from north america --limit 2` selects one probe in USA **and** one in Canada.
 
-- `from cogent+europe+datacenter` will return a probe hosted with Cogent, located in Europe and tagged as a datacenter probe. Or you could use `eyeball` to do the opposite.
+## Best practices and tips
+Learn to use Globalping in the most optimal way!
 
-### Multiple locations
+### Test with "magic" 🧙
+All integrations connect to the same API and use our "magic" field as location input.
+This maintains a consistent and straightforward user experience, allowing you to reuse logic and parameters across all tools.
 
-To select multiple locations in a single request you can use a comma. And of course you can combine filters and multiple locations at the same time.
+So, for example, when you run a test from "Germany" using the CLI or Slack app, our API's magic parser processes the location.
```

Comment:
I think I've seen you mention only the CLI and Slack, but the Web UI uses the same magic field too, same as the bot and all others. Just wanted to make it clear.